packages/postcss-reduce-initial/src/index.js

```diff
@@ -1,18 +1,22 @@
 'use strict';
 const browserslist = require('browserslist');
 const { isSupported } = require('caniuse-api');
-const fromInitial = require('../data/fromInitial.json');
-const toInitial = require('../data/toInitial.json');
+const fromInitial = require('./data/fromInitial.json');
```

Comment:
Need to move the data files because TypeScript needs each subproject in a composite build to be inside a single root.